test/tests/performance/pbs_equiv_classes_perf.py

```diff
@@ -51,19 +51,41 @@ def setUp(self):
         TestPerformance.setUp(self)
         self.scheduler.set_sched_config({'log_filter': 2048})
 
-    @timeout(1000)
+        # Create vnodes
+        a = {'resources_available.ncpus': 1, 'resources_available.mem': '8gb'}
+        self.server.create_vnodes('vnode', a, 10000, self.mom,
+                                  sharednode=False)
+
+    def get_cycle_time(self):
+        """
+        Calculate the time between scheduling cycles
```

Comment:
You aren't calculating the time between cycles, you are running a cycle and then calculating the time of that cycle.